src/main/java/seedu/todo/commons/util/TextAreaResizerUtil.java

```diff
@@ -0,0 +1,63 @@
+package seedu.todo.commons.util;
+
+import javafx.beans.value.ChangeListener;
+import javafx.scene.control.TextArea;
+import javafx.scene.text.Text;
+
+import java.util.Timer;
+import java.util.TimerTask;
+
+
+/**
+ * Automatically resize a text area object such that the height fits the content of the text area
+ */
+public class TextAreaResizerUtil {
+
+    private Text mockText; //A text object that will imitate the text in the TextArea object
+    private double singleLineHeight;
+
+    /**
+     * Set a particular TextArea object to have it's height automatically resize-d.
+     */
+    public void setResizable(TextArea textArea) {
+        bindToMockText(textArea);
+        setupTextArea(textArea);
+    }
+
+    private void bindToMockText(TextArea textArea) {
+        mockText = new Text();
+        mockText.textProperty().bind(textArea.textProperty());
+        mockText.fontProperty().bind(textArea.fontProperty());
+        mockText.wrappingWidthProperty().bind(textArea.widthProperty());
+    }
+
+    private void setupTextArea(TextArea textArea) {
+        ChangeListener<Object> changeListener = (observable, oldValue, newValue) -> {
+            double height = getCorrectedHeight() + 10;
```

Comment:
Tried, and CSS padding won't work because height is dynamically computed, and CSS padding will squash everything